webpack.prod.config.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
```

Comment:
I'm not sure that this is needed. ES2015 is always in strict mode?

Reply:
*extremely unsure about js versions voice*

How do we flag the js version in specific files? [We lint against es5 across all of edx-platform AFAICT](https://github.com/edx/edx-platform/blob/master/.eslintrc.json#L2). *Is* this file ES2015?

Reply:
This file isn't exactly ES2015. It's JS that has to be able to run in Node.js. [Node.js has been adding lots of support for ES2015 features](http://node.green/) but it doesn't support everything, for example the `import ... from ...` style ES modules. In order for us to use features like that we'd have to run Babel on the config to transpile it to something Node.js can run before actually running Webpack (which seems a bit too convoluted to me).

ES versions are confusing because the standards: ES5, ES2015, ES2016, ES2017 are mainly specs (with some reference implementations). Every Javascript engine, [of which there are many](https://en.wikipedia.org/wiki/ECMAScript#Implementations), that runs JS has to individually add support for the new features. So engines are usually somewhere in-between standards on what they support. Node.js uses the same engine (V8) as Google Chrome, so you can think of it as the same environment as running raw JS in the browser without running it through Babel first. V8 actually supports a lot of the ES2015 features, just not *everything*.

Seeing as Babel adds `"use strict";` to the output bundles, we should probably have it here since this file isn't run through Babel.

Reply:
That was incredibly informational, thanks @thallada!